src/plugins/analysis/device_tree/code/device_tree.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from common_helper_process import execute_shell_command_get_return_code
+
+from analysis.PluginBase import AnalysisBasePlugin
+
+
+class AnalysisPlugin(AnalysisBasePlugin):
+    '''
+    Device Tree Plug-in
+    '''
+    NAME = 'device_tree'
+    DESCRIPTION = 'get the device tree in text from the device tree blob'
+    DEPENDENCIES = []
+    VERSION = '0.1'
+    # MIME_WHITELIST = ['linux/device-tree']
```

Comment:
Why is this commented out?

Reply:
I commented it out for testings, but forgot  to undo it...